docs/usersguide/data.rst

```diff
@@ -0,0 +1,43 @@
+.. _usersguide_data:
+
+============
+Atomic & Isotopic Data
+============
+
+.. currentmodule:: pyne.data
+
+.. automodule:: pyne.data
+
+For a complete specification for the classes in the ``cccc`` module, please
+refer to the Library Reference entry for :ref:`pyne_cccc`.
+
+***************************
+Example Use of Data Class
+***************************
+
+The isotopic & elemental abundance data is usually loaded at the time the atomic_mass function is launched. This data now
+exists in the C++ implementaion, behinds the scenes the nuc_data.h5 file is loaded and the appropriate data elements populated.
+
+
+.. code-block:: ipython
+
+   In [1]: from pyne.data import atomic_mass
+
+   In [2]: print atomic_mass('2H')
+   2.01410177812
+
+If one chooses to, the nuclear data path can be set to some nonesense value and the data will be initialised from memory
+instead.
+
+.. code-block:: ipython
+
+   In [1]: from pyne.pyne_config import pyne_conf
+   
+   In [2]: from pyne.data import atomic_mass
+   
+   In [3]: pyne_conf.NUC_DATA_PATH = b'some silly path that doesnt exist'
```

Comment:
I see that as bad behavior on the part of the function trying to open the file at the location specified by `pyne_conf.NUC_DATA_PATH`.  Maybe you could add the conditional: `if isinstance(pyne_conf.NUC_DATA_PATH, str)` to said function.